### PR TITLE
Bugsnag vapor 3

### PR DESCRIPTION
**Pull request by BrettRToomey**

### CODE REVIEW

File: source/_posts/2019-01-14-Google-Home.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: A Guide to Google Home Integrations
+date: 2019-01-14 10:00:00
+tags: server,google,home,ai,api,cloud,serverless
+authorIds: 
+- brto
+categories:
+- Vapor
+---
+
+A while back I was tasked with integrating DMI’s weather service into Google Home. The goal of the service was to  enable users to ask:  “Hey DMI, what is the weather.” As well as: “Hey DMI, should I bike to work today?” Given my only experience with Google Home was through its advertisements, I went in with the expectations that integration would be quick and painless—just configure a few interests, link it with DMI’s API and start checking the weather. Surprisingly, this was far from actuality. Configuration of the app, dialogues and Cloud Functions is split across 3 different services. The initial clunkiness aside, the services seemed to work well and the tooling was relatively stable. Training intents in DialogFlow was a seamless and enjoyable experience, it just worked. Using the website’s inline code editor, I was able to develop and deploy the fulfillment to Google Cloud Functions.
```
Comment (pbodsk) [inline source/_posts/2019-01-14-Google-Home.md]: Do people outside Denmark know what DMI is? Maybe just a link and a short mention of what it is the first time you mention it

File: source/_posts/2019-01-15-Bugsnag.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+date: 2019-01-15 10:00:00
+tags: server,bugsnag,report,reporting,release
+authorIds: 
+- brto
+categories:
+- Vapor
+---
+
+At Nodes we have been working hard on a Vapor 3 provider for the [Bugsnag](https://www.bugsnag.com) reporting tool. Bugsnag is a reporting tool that we use extensively to help catch and fix bugs.
+
```
Comment (pbodsk) [inline source/_posts/2019-01-15-Bugsnag.md]: you mention reporting tool two times in a row, maybe

> At Nodes we have been working hard on a Vapor 3 provider for [Bugsnag](https://www.bugsnag.com). Bugsnag is a reporting tool that we use extensively to help catch and fix bugs.

File: source/_posts/2019-01-15-Bugsnag.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+In the new version of the library, we've been able to remove a lot of dead code, simplify the API and add support for breadcrumbs.
+
+## Migration to Vapor 3
+
+The migration from Vapor 2 to Vapor 3 was a relatively easy one for Bugsnag. The major changes were the use of `Codable` and the introduction of `Future`s.
+
+One of the double-edged-swords of Vapor 3 is the asynchronous design. The asynchronous nature of the framework makes stack traces virtually useless. Sadly, we had to remove support for CStack and stack traces.
```
Comment (pbodsk) [inline source/_posts/2019-01-15-Bugsnag.md]: What is CStack? Or is it just me being the ignorant frontender here?

Comment (pbodsk) [reply source/_posts/2019-01-15-Bugsnag.md]: was it the one you wrote a blog post about at some point?

Comment (BrettRToomey) [reply source/_posts/2019-01-15-Bugsnag.md]: Yeah

Comment (pbodsk) [reply source/_posts/2019-01-15-Bugsnag.md]: Maybe a link to that blog post?

**Pull request merged**